.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,7 +13,7 @@ Fixes [list issues/bugs if needed]
 
 ### Check if PR is ready for release
 
-If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:
+If this PR contains Vanilla SCSS or macro code changes, it should contain the following changes to make sure it's ready for the release:
```

Comment:
There are more specific CSS version guidelines below, which probably could be updated to mention macros as well (as in: if macro API features are added: minor release, if API changes: major).

Reply:
@bartaz  Thanks, I've tweaked it further to more comprehensively cover macros & CSS class changes. Let me know what you think.